### PR TITLE
fix: update apple guide

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/social-signin/30_apple.mdx [hunk 1]
```diff
@@ -70,7 +93,7 @@ Follow these steps to add Apple as a social sign-in provider to your project usi
 
 Follow these steps to add Apple as a social sign-in provider to your project using the Ory CLI:
 
-1. Using an Apple Developer Account, create an app, a service, and a private key.
+1. Using an Apple Developer Account, create an app, a service, and a private key. Check the Ory Console guide for the concrete steps.
```
Comment (vinckr) [inline docs/kratos/social-signin/30_apple.mdx:96]: not a blocker IMO but not ideal to be honest. 

we maybe need to refactor the whole section here - its really a pita to edit these and we get more out of sync with every edit. 

then again its not really broken (yet)

File: docs/kratos/social-signin/30_apple.mdx [hunk 2]
```diff
@@ -27,9 +27,32 @@ Follow these steps to add Apple as a social sign-in provider to your project usi
...
+   3. Enter a description and bundle ID of your liking.
+   4. Scroll down and select **Sign in with Apple**.
+   5. Click **Continue**, then **Register**.
+   6. Go back to the [Identifiers overview](https://developer.apple.com/account/resources/identifiers/list/serviceId).
+   7. Next to the search icon open the dropdown and select "Services IDs".
+   8. Create a new [Services ID](https://developer.apple.com/account/resources/identifiers/add/serviceId).
+   9. Choose a description of your liking. The identifier **must match** the App ID value.
+   10. Click **Continue**, then **Register**.
+   11. Click on the newly created service identifier and click the "Sign in with Apple" checkbox.
+   12. Click the **Configure** button and set the domains and subomain to match your Ory Network domain (or custom hostname).
```
Comment (vinckr) [inline docs/kratos/social-signin/30_apple.mdx:42]:
```suggestion
   12. Click the **Configure** button and set the domains and subdomain to match your Ory Network domain (or custom hostname).
```

File: docs/kratos/social-signin/30_apple.mdx [hunk 3]
```diff
@@ -27,9 +27,32 @@ Follow these steps to add Apple as a social sign-in provider to your project usi
...
+   13. Add the Redirect URI you received earlier and set it here as the return URL and click **Next**.
+   14. Click **Continue**, then **Save**.
+   15. Next, go to [**Keys**](https://developer.apple.com/account/resources/authkeys/list) and [register a new key](https://developer.apple.com/account/resources/authkeys/add).
+   16. Set a key name and enable **Sign in with Apple**. Next to Sign in with Apple, click **Configure**.
+   17. Use the App ID you created earlier as the primary AppID.
+   18. Click **Continue**, then **Register**.
+   19. **Download** the key and click **Done**.
+5. Copy the correct identifiers to the Ory Console Apple configuration:
+   * Client ID: Add the identifier of the Services ID (**not** the Bundle ID) you created in Apple. Not the ID of the App ID. Not the Team ID.
+   * Client Secret Signing Key: Paste the contents of your keyfile downloaded from Apple. Paste the entire key, including the BEGIN/END PRIVATE KEY lines.
```
Comment (vinckr) [inline docs/kratos/social-signin/30_apple.mdx:52]:
```suggestion
   * Client Secret Signing Key: Paste the contents of your key file downloaded from Apple. Paste the entire key, including the BEGIN/END PRIVATE KEY lines.
```

File: docs/kratos/social-signin/30_apple.mdx (hunk 1 above)
Comment (aeneasr) [reply docs/kratos/social-signin/30_apple.mdx:96]: Yeah, we can copy paste it over. But I was a bit skeptical over how to do it. Maybe we just put it in the sections before or link to a section? Suggestions welcomed.

Comment (vinckr) [reply docs/kratos/social-signin/30_apple.mdx:96]: I think it is fine like this, its a lot of "busy" work to include it in two sections IMO and the structure makes my head hurt. 
I just wanted to comment that we can maybe find a better layout for these guides altogether.

I am thinking 
- howto that is specific to the provider
- and then a short description how you do it in CLI and console that we can copy to all providers
- if needed: a section that explains some special things about this particular provider

This way you only need the steps once, and easy to keep in sync and add new providers.
 
Its probably a couple hours work to update all the guides for this, so I would not put it super high priority but can be done when we clean up the docs a bit, 
WDYT?

**Review by vinckr — APPROVED** [review]
lgtm :)

**Pull request merged**